profiles/sssd/nsswitch.conf

```diff
@@ -1,6 +1,6 @@
-passwd:    files sss
+passwd:    sss files
 shadow:    files sss
```

Comment:
You don't have to add sss to shadow at all, because we don't implement the shadow map. basically, look at what nsswitch.conf contains in F-27..